docs/get-started/lotus/installation.md

```diff
@@ -311,19 +335,29 @@ We recommend that macOS users use [Homebrew](https://brew.sh) to install each of
 
    This is due to a Lotus bug that prevents Lotus from running on a processor without `adx` instruction support, and should be fixed soon.
 
-1. Build Lotus:
+5. Build and install Lotus
 
-   ```sh
-   make clean && make all # mainnet
+   Lotus is compiled to operate on a single network,  run one of the following commands to build the lotus node for the specific lotus network.
```

Comment:
```suggestion
   Lotus is compiled to operate on a single network. Run one of the following commands to build lotus for your intended network.
```

A few nit-picks: 
1. Separate the two sentences with a period.
2. Networks are not lotus-specific; you can use any implementation (venus, forest, etc.) to connect to them.
3. Once the lotus application is built, you can use it to kick off multiple nodes. So technically speaking you're not building a lotus node.